core/src/main/java/nu/yona/server/goals/entities/TimeZoneGoal.java

```diff
@@ -43,4 +45,11 @@ public boolean isNoGoGoal()
 	{
 		return false;
 	}
+
+	@Override
+	public boolean isGoalAccomplished(DayActivity dayActivity)
+	{
+		// TODO: zones should be parsed? maybe in spread format
+		return true;
```

Comment:
Maybe it is easier to use 96 `boolean`s to model the allowed time zones instead of just `String`s. If `String` is still preferred, we need to establish a fixed format that can be parsed, e.g. "HH:mm-HH:mm".